dags/looker.py

```diff
@@ -172,8 +172,8 @@
             "pip install spectacles==2.4.10 && " # todo: remove this once mozilla-nimbus-schemas supports newer pydantic version
             "spectacles content --verbose"
             " --base-url ${BASE_URL}"
-            " --client-id ${LOOKER_API_CLIENT_ID}"
-            " --client-secret ${LOOKER_API_CLIENT_SECRET}"
```

Comment:
I'd think you'd no longer want to pass these secrets below as `secrets=[looker_api_client_id_prod, looker_api_client_secret_prod]` below given this change, right?